pkgs/applications/graphics/potreeconverter/default.nix

```diff
@@ -29,8 +29,16 @@ stdenv.mkDerivation rec {
   ];
 
   patchPhase = ''
+    runHook prePatch
+
     substituteInPlace ./CMakeLists.txt \
       --replace "find_package(TBB REQUIRED)" ""
+
+    # prevent inheriting permissions from /nix/store when copying
+    substituteInPlace Converter/src/main.cpp --replace \
+      'fs::copy(templateDir, pagedir, fs::copy_options::overwrite_existing | fs::copy_options::recursive)' 'string cmd = "cp --no-preserve=mode -r " + templateDir + " " + pagedir; system(cmd.c_str());'
```

Comment:
@MatthewCroughan  I think there should be at least a comment indicating why with this `system()` invocation, shell injection is impossible. Is it guaranteed?